packages/web/components/table/cells/market-cap-cell.tsx

```diff
@@ -0,0 +1,37 @@
+import { Dec } from "@keplr-wallet/unit";
+import classNames from "classnames";
+import { observer } from "mobx-react-lite";
+import { FunctionComponent } from "react";
+
+import SkeletonLoader from "~/components/skeleton-loader";
+import { AssetCell as Cell } from "~/components/table/cells/types";
+import { useStore } from "~/stores";
+
+export const MarketCapCell: FunctionComponent<Partial<Cell>> = observer(
+  ({ coinDenom }) => {
+    const { queriesExternalStore } = useStore();
+    const tokenDataQuery = queriesExternalStore.queryTokenData.get(coinDenom!);
+
+    const dayChange = tokenDataQuery.get24hrChange;
```

Comment:
Do you still need the market cap? This is 24h change

Reply:
It's defined on the next PR #2191. It's just a placeholder on this one